### ISSUE TITLE
Promtail scraping isn't scraping logs and stays outdated by a minimum 2 days

### ISSUE THREAD

**Issue by 5aberd2** [grafana/loki#7691]
Description:
I’m having a problem with one log in promtail for example I have two logs (Syslog, custom.log)
syslog populates/scrapes to the latest minute but the other (used by the docker container) is not and is 3 days old although there is new logs in the file, what could be the problem in this case?

**Expected behavior**
it should show the recent changes in the logs

Docker Promtail
Docker Loki
Both are on Ubuntu Host 18.04

**Comment by OsmelLavalier** [comment]
Im also facing the same problem. Any updates?

**Comment by 5aberd2** [comment]
Nope actually nothing for now I'm restarting the container every two days until someone view this as an issue (lol)

**Comment by OsmelLavalier** [comment]
Fixed it, for me it was a problem of permissions :)

**Comment by 5aberd2** [comment]
> Fixed it, for me it was a problem of permissions :) 

Cool, permissions of what exactly? Of the log file itself?

**Comment by OsmelLavalier** [comment]
> f the log file itself?

Right, I was referring to my custom log file was not being catched by my docker service. Also check for the scrape_configs in promtail I some troube with that as well. Maybe that helps. It would be nice if someone from Loki also responded. Because it was not clear to my why my solution fixed it

**Comment by 5aberd2** [comment]
Got it, yeah I'll have to try it or if someone from loki helped.

Best Regards,
Ali Awwad
________________________________

**Comment by JStickler** [comment]
Promtail is end of life (EOL) as of  March 2, 2026.  No future support or updates will be provided. All future feature development will occur in [Grafana Alloy](https://github.com/grafana/alloy).
